charts/delphai-deployment/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: "v2"
+name: "delphai-deployment"
```

Comment:
Universal name describing it's a core k8s type, not some unused feature as it was with `keda`